api/v1beta1/verticadb_types.go

```diff
@@ -496,6 +505,21 @@ type Subcluster struct {
 	ExternalIPs []string `json:"externalIPs,omitempty"`
 }
 
+type Affinity struct {
```

Comment:
Lets add a comment to explain why this exists and why we don't use the Kubernetes one.